website/project/model.py

```diff
@@ -2300,6 +2303,13 @@ def update_node_wiki(self, name, content, auth):
         )
         new_page.save()
 
+        wiki_changed.send(
+            name,
```

Comment:
Not quite sure what you mean here @sloria, the issue is that the send seems not to like anything other than a string in this position. At least from my understanding, please enlighten me.